Tickets/sitcom 1223

ltoribiosc opened the pull request.

Commit:
Create SITCOM-1223-TMA_Capacitor Bank discharge vs. Acceleration profiles

Test verifying successful push

compareIMSMount function

Init computeSettleTrack function

plot of computeSettleTrack function

instability detection in M3M1

first event classification

rms

tracking classification for RMS

proof

version to review

delete SITMCOMTN-XXX

version to review

upload SITCOM-1136

comments of ReviewNB (part 2)

Solve: RuntimeError: Found multiple blockIds ({'BL13_O_20231219_000001', 'BL13_O_20231220_000001'}) for seqNum=1

Commit:
Create functions for the analysis

Commit:
Upload the Functions

Commit:
Calculation slew-error for small angles

Commit:
Ploting power supply voltage vs Az velocity

Plot power supply voltage vs Az velocity and test

Commit:
Function to create an azimuth histogram

Commit:
Update plot_spv_vs_dist with histogram

Commit:
Analysis simultaneous movements in az and el

Analysis of Az and El moved at same time

Commit:
Add comments to the code

Commit:
Testing the code with different nights observations

Commit:
Testing errors

Commit:
Upload get_acc_jerk function

bstalder reviewed (changes requested):
please remove the 1136 file changes.  also clear outputs to the 1223 notebook.  also inline plots are preferred instead of file outputs.

ltoribiosc committed:
Clean outputs

copy last version SITCOM-1136

ltoribiosc commented:
Thanks Brian. I just deleted the notebook outputs and deleted the 1136 files.

bstalder commented:
think you need further commits.  I still see the M1M3 notebook with changes.

ltoribiosc commented:
I think you need more commits. I still see the M1M3 notebook with changes.


Where do you see the M1M3 notebook changes? I deleted all the files that were copied from the previous ticket. You should see nothing now. 

I now see 13 commits all related to SITCOM-1223. That is, from this ticket

bstalder commented:
I still see it under "file changes"

ltoribiosc commented on notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb:
This file was introduced due to an error when creating the branches. The SITCOM-1223 branch was not created from develop branch but from SITCOM-1136 by mistake. Therefore, in order to merge them it was necessary to copy the last files of SITCOM-1136 here

ltoribiosc reviewed (commented):
SITCOM-1136 changes have been reviewed. This file was introduced due to an error when creating the branches. The SITCOM-1223 branch was not created from develop branch but from SITCOM-1136 by mistake. Therefore, in order to merge them it was necessary to copy the last files of SITCOM-1136 here

ltoribiosc committed:
Delete notebooks/tel_and_site/subsys_req_ver/m1m3/SITCOM-1136_M1M3_analysis_stability_tracking.ipynb

We have deleted the SITCOM-1136 file because it is not needed for the SITCOM-1223 branch.

Signed-off-by: Laura Toribio San Cipriano <144215139+ltoribiosc@users.noreply.github.com>
